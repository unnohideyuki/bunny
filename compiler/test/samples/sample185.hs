x :: Float
x = 123456789012345678901234567890 :: Float

y :: Float
y = 3

main = do print (x + y)
          print (x - y)
          print (x * y)
          print (signum (1 :: Float))
          print (signum (0 :: Float))
          print (signum (-1 :: Float))
