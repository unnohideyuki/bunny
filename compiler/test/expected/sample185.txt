1.2345679e29
1.2345679e29
3.7037037e29
1.0
0.0
-1.0
